Swiften/Parser/PayloadParsers/RosterParser.cpp

```diff
@@ -52,11 +52,15 @@ void RosterParser::handleStartElement(const std::string& element, const std::str
             if (attributes.getAttribute("ask") == "subscribe") {
                 currentItem_.setSubscriptionRequested();
             }
+        } else if (element == "annotate" && ns == "urn:xmpp:mix:roster:0") {
+            getPayloadInternal()->setAnnotate(true);
```

Comment:
If this is MIX-specific, shouldn't it be obvious from the method name? setSupportsMIX or setSupportsMIXAnnotations or something?